bundle/libraries/upload.go

```diff
@@ -153,13 +162,7 @@ func (u *upload) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
 	errs.SetLimit(maxFilesRequestsInFlight)
 
 	for _, source := range sources {
-		relPath, err := filepath.Rel(b.SyncRootPath, source)
-		if err != nil {
-			relPath = source
-		} else {
-			relPath = filepath.ToSlash(relPath)
-		}
-		cmdio.LogString(ctx, fmt.Sprintf("Uploading %s...", relPath))
+		cmdio.LogString(ctx, fmt.Sprintf("Uploading %s...", source))
 		errs.Go(func() error {
 			return UploadFile(errCtx, source, u.client)
```

Comment:
Before UploadFile was taking absolute path if I'm not mistaken and now it takes normalised one (relative + forward slashes). How does it make UploadFile open the file later for upload?